README.md

```diff
@@ -326,7 +326,8 @@ maintenance and customization.
 
  - **05.adi-tools** - this stage installs the following ADI tools: libiio, pyadi, libm2k, libad9361, libad9166,
    iio-oscilloscope, iio-fm-radio, fru_tools, jesd-eye-scan-gtk, colorimeter, Scopy, Gnuradio and gr-m2k based on settings from config. 
-   A file with all the tools installed, their branches and SHAs is created in the same stage.
+   By default the linux_image_ADI-scripts repository is cloned and a file with all the tools installed, their branches and SHAs
```

Comment:
Maybe you can add quotation around repo name, and reformulate a bit "a file with all the tools installed" - it weird to say that a file is installed, can be replaces with 'added', 'created' or something similar.

Reply:
Should i also add quotations around the other repo names (libiio, pyadi, etc)?